src/index.ts

```diff
@@ -56,9 +69,21 @@ class Bot {
    * bot.initFromRunningService()
    */
   public async initFromRunningService(homeDir?: string, options?: InitOptions): Promise<void> {
+    this._beginInitState()
     await this._prepWorkingDir()
+    await this._maybePrepDebugDir(options)
     await this._service.initFromRunningService(homeDir, options)
     await this._initSubBots(options)
+    await this._maybeLogCopyLoop(options)
+    this.debugLog('initialized')
+  }
+
+  private _beginInitState(): void {
+    if (this._initStatus !== 'preinit') {
+      throw new Error(`tried to init, but state is already ${this._initStatus}`)
```

Comment:
Do we want to allow a deinited bot to be reinitialized? I think I remember being able to do that, but this check no longer allows that.

Reply:
no, I removed that on purpose and changed the test case. I think it's dangerous, and no one really needs to do it in practice. While I don't think we have any bugs in our library, it seems a bit weird to me to have the bot possible to cross from one user to another. Safer to instantiate a new one.